web/ui/react-app/src/App.css

```diff
@@ -10,6 +10,26 @@ button.classic-ui-btn {
     margin-bottom: 20px;
 }
 
+.navbar a {
```

Comment:
I think none of these custom CSS styles should be needed, as we just want the default Bootstrap dark navbar (https://getbootstrap.com/docs/4.3/components/navbar/#color-schemes). When I remove the custom styles and add the `navbar-dark` class to the navbar, it starts looking (almost) like the one in the classic UI.

Reply:
Yep, you are right. "dark" replace all custom CSS, except the active state of the tabs. I'll check whether reactstrap doesn't provide something to replace the isActive wrapper from the NavBar